install.sh

```diff
@@ -37,7 +37,8 @@ else
   echo "user provided as arg = $user"
 fi
 
-sudo apt-get install -y  bluetooth libbluetooth-dev
+sudo apt-get install -y bluetooth libbluetooth-dev
+sudo apt-get install -y python3-venv
 
 sudo mkdir -p /opt/v
```

Comment:
Would it make sense to place this under the emonhub `script_dir` to keep it all together?

Reply:
dont understand what you mean

Reply:
You are putting the venv directory at what I suggest is a random location. Put it under the variable 'emon_script' location so it keeps everything together.

Reply:
by random location, you mean `/opt/v` ?
maybe yes another name than v could be better, something like `/opt/venv` so the virtual env for emonhub would be in `/opt/venv/emonhub` and not in `/opt/v/emonhub`
maybe you mean `/opt/openenergymonitor/venv` ?